.gitignore

```diff
@@ -6,6 +6,8 @@ test-output/
 .sbtopts
 project/.sbt
 test-output/
+.bloop
```

Comment:
I think that good place for this is in  ~/.config/git/ignore . Likewise for `.idea/**`, `*.ipr`, etc.

Reply:
I already have that in my ~/.config thought some value having it there for anyone who doesn't. metals and bloop are so popular now in my opinion it should be part of any default gitignore